models/git/protected_branch.go

```diff
@@ -38,17 +38,22 @@ type ProtectedBranch struct {
 	isPlainName                   bool                   `xorm:"-"`
 	CanPush                       bool                   `xorm:"NOT NULL DEFAULT false"`
 	EnableWhitelist               bool
-	WhitelistUserIDs              []int64  `xorm:"JSON TEXT"`
-	WhitelistTeamIDs              []int64  `xorm:"JSON TEXT"`
+	WhitelistUserIDs              []int64 `xorm:"whitelist_user_ids JSON TEXT"`
+	WhitelistTeamIDs              []int64 `xorm:"whitelist_team_ids JSON TEXT"`
```

Comment:
existing names can't be changed, otherwise it breaks exiting instances.

```suggestion
	WhitelistUserIDs              []int64 `xorm:"JSON TEXT"`
	WhitelistTeamIDs              []int64 `xorm:"JSON TEXT"`
```